Makefile

```diff
@@ -49,10 +48,8 @@ run:
 clean:
 	@rm -rf $(TARGET_DIR)
 
-# image: imagebuilder
-# 	@if [ $${USE_IMAGE_STREAM:-false} = false ] && [ $${SKIP_BUILD:-false} = false ] ; \
-# 	then $(IMAGE_BUILDER) -t $(IMAGE_TAG) . $(IMAGE_BUILDER_OPTS) ; \
-# 	fi
+image: $(info Building Docker image...)
+	@docker build -f $(DOCKERFILE) -t $(IMAGE_TAG) $(CURPATH) $(IMAGE_BUILD_OPTS)
```

Comment:
I think we may give develops choice here, so they can chose to use which tool to build the image, such as podman, buildah or docker.